apps/web/playwright/fixtures/users.ts

```diff
@@ -319,8 +321,22 @@ export const createUsersFixture = (page: Page, emails: API | undefined, workerIn
               }),
               store.page
             );
+            teamMatesIds.push(teamUser.id);
             store.users.push(teammateFixture);
           }
+          // Add Teammates to OrgUsers
+          if (scenario.isOrg) {
```

Comment:
this is just a suggestion for the future not a change requirement, since we seem to be adding more and more scenarios to this function using if statements, we might want to consider soon refactoring this into smaller functions and using a switch cases when possible.